Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ruby:2.7.6
+FROM ruby:3.2.0
```

Comment:
Any reason not to use 3.2.2 or 3.2 (i.e. latest security patch)?

Reply:
Not sure, just that I've tested it against ruby 3.2.0 and have sorted out the dependencies so don't want to mess around with it any more

Reply:
No probs 👍 Should be a drop-in replacement so hopefully a very simple update when you have a spare moment!